UI/html/index.html

```diff
@@ -17,7 +17,8 @@
     <div class="container">
       <div id="branding">
         <a href="index.html">
-          <img src="../images/bus-logo.png" alt="">
+          <!-- <img src="../images/bus-logo.png" alt=""> -->
+          <!-- <img src="https://img.icons8.com/clouds/75/000000/bus.png"> -->

```

Comment:
Get rid of this comment code.

Reply:
Okay
I'm working on it and I will merge now.
Thank you